frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeContext.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Outlet } from 'react-router-dom';
+import {
+  Bullseye,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Spinner,
+  Title,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { TemplateKind } from '~/k8sTypes';
+import { DEFAULT_CONTEXT_DATA } from '~/utilities/const';
+import { ContextResourceData } from '~/types';
+import { useContextResourceData } from '~/utilities/useContextResourceData';
+import useTemplates from './useTemplates';
+
+type CustomServingRuntimeContextType = {
+  refreshData: () => void;
+  servingRuntimeTemplates: ContextResourceData<TemplateKind>;
+};
+
+export const CustomServingRuntimeContext = React.createContext<CustomServingRuntimeContextType>({
+  refreshData: () => undefined,
+  servingRuntimeTemplates: DEFAULT_CONTEXT_DATA,
+});
+
+const CustomServingRuntimeContextProvider: React.FC = () => {
+  const servingRuntimeTemplates = useContextResourceData<TemplateKind>(useTemplates('opendatahub')); // TODO: change to operator namespace
+
+  const servingRuntimeTeamplateRefresh = servingRuntimeTemplates.refresh;
+
+  const refreshData = React.useCallback(() => {
+    servingRuntimeTeamplateRefresh();
+  }, [servingRuntimeTeamplateRefresh]);
+
+  if (servingRuntimeTemplates.error) {
+    return (
+      <Bullseye>
+        <EmptyState>
+          <EmptyStateIcon icon={ExclamationCircleIcon} />
+          <Title headingLevel="h4" size="lg">
```

Comment:
A11y issue

Reply:
Hi @andrewballantyne can you help me out here? Where's the A11y in this piece of code, I was reviewing all the latest fixes and I couldn't figure out the issue here.

Reply:
Ok great! yeah thank you! I wasn't 100% sure of that, changing it now!